pkg/applicationserver/io/packages/loradms/v1/api/objects/objects.go

```diff
@@ -79,14 +81,29 @@ func (d DeviceUplinkResponses) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// PositionSolution is the solution of a position calculation.
+type PositionSolution struct {
+	ECEF           []float64 `json:"ecef"`
+	LLH            []float64 `json:"llh"`
+	CaptureTimeGPS float64   `json:"capture_time_gps"`
+	GDOP           float64   `json:"gdop"`
+	Accuracy       float64   `json:"accuracy"`
+	Timestamp      float64   `json:"timestamp"`
+}
+
 // UplinkResponse contains the state changes and completed items due to an uplink message.
 type UplinkResponse struct {
-	File              *File            `json:"file"`
-	StreamRecords     []Stream         `json:"stream_records"`
-	FullfiledRequests []PendingRequest `json:"fulfilled_requests"`
-	Downlink          *LoRaDnlink      `json:"dnlink"`
-	InfoFields        InfoFields       `json:"info_fields"`
-	LogMessages       []LogMessage     `json:"log_messages"`
+	File                  *File             `json:"file"`
+	StreamRecords         []Stream          `json:"stream_records"`
+	PositionSolution      *PositionSolution `json:"position_solution"`
+	PendingRequests       *PendingRequests  `json:"pending_requests"`
+	FullfiledRequests     []Request         `json:"fulfilled_requests"`
+	Downlink              *LoRaDnlink       `json:"dnlink"`
+	InfoFields            InfoFields        `json:"info_fields"`
+	LogMessages           []LogMessage      `json:"log_messages"`
+	UploadedFiles         []File            `json:"uploaded_files"`
+	UploadedStreamRecords []Stream          `json:"uploaded_stream_records"`
+	LastUplink            *LoRaUplink       `json:"last_uplink"`
 }
```

Comment:
@adriansmares : Some fields are not a part of the `UPLINK_RESPONSE` object defined [here](https://www.loracloud.com/documentation/device_management?url=v1.html#uplink-response):
- `UploadedFiles`
- `UploadedStreamRecords`
- `LastUplink`

Then you correctly spotted issues in the documentation:

- `StreamRecords` is returning a `STREAM_UPDATE` array, doc [here](https://www.loracloud.com/documentation/device_management?url=v1.html#stream-update)
- `FullfiledRequests` should be named `PendingRequests` 
- `PendingRequests` should be added does not return a `[pending_request]` array but a `[request]` array


`FullfiledRequests` example:
```
 "fulfilled_requests":[
        {
          "type":"GETINFO",
          "param":[
            "signal",
            "temp"
          ]
        }
      ],
```

`pending_request` return by the DAS without requests (empty `requests` array):
```
      "pending_requests":{
        "requests":[        ],
        "id":2,
        "updelay":20,
        "upcount":1
      },
```

`stream_records` example:
```
      "stream_records":[
        [
          0,
          "010203"
        ]
      ],
```